gensrc/proto/segment_v2.proto

```diff
@@ -151,6 +151,12 @@ message ColumnMetaPB {
     repeated ColumnIndexMetaPB indexes = 8;
     // pointer to dictionary page when using DICT_ENCODING
     optional PagePointerPB dict_page = 9;
+
+    repeated ColumnMetaPB children_columns = 10;
+    repeated string children_column_names = 11;
```

Comment:
I think we don't need `children_column_names ` and `num_rows`,  and in your code , smees you don't use them.

Reply:
`children_column_names` can be remove. 
`num_rows ` is used in `OrdinalIndexReader`

Reply:
OK